### PR TITLE
Adding lsdc-edna conda env

### PR DESCRIPTION
**Pull request by vshekar**

### CODE REVIEW

**Comment by JunAishima** [comment]
I expect this will be a low turnover environment that will hopefully get replaced when we change to using a new analysis program. that is one reason I didn't want to create another repo (you mean like lsdc-server?)

**Comment by mrakitin** [comment]
The CI won't produce anything for this environment. We need to have it in a separate repo (one type of env per repo).

**Comment by JunAishima** [comment]
I just built it locally - this is ok for now.

**Comment by mrakitin** [comment]
@JunAishima, @vshekar, question - why is Python 2.7 required? If it's only the `fabio` package v0.4.0, it's available in conda-forge for Python 3.6.

**Comment by mrakitin** [comment]
Ah, I see, I did not realize this package is used there too. How do you incorporate it into the environment?

**Comment by mrakitin** [comment]
Sounds good, thanks for the explanation, @JunAishima!

File: configs/lsdc-edna.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+docker_image: "nsls2/linux-anvil-cos7-x86_64:latest"
```
Comment (mrakitin) [inline configs/lsdc-edna.yml:1]: @vshekar, do you want to switch to conda-forge's image here?

Comment (JunAishima) [reply configs/lsdc-edna.yml:1]: please use the change from the following pull request as a model: https://github.com/nsls2-conda-envs/conda-pack-template/pull/73

Comment (vshekar) [reply configs/lsdc-edna.yml]: Switched in 4fcf02473adf373598ce628a8c11c429eb6b5bd3

**Review by mrakitin — APPROVED** [review]

**Pull request merged**